### PR TITLE
Add new building

### PR DESCRIPTION
**Pull request by zsofi75**
Hi dears, please check and let it be merged if everything is ok.

### CODE REVIEW

File: Tribes/src/app/dashboard/dashboard.component.css [hunk 1]
```diff
@@ -43,3 +43,23 @@ a :hover {
 	color: #212529;
     background-color: #ffc107;
 }
+
+.badge-add {
+    border-radius: 14px;
```
Comment (balazsbarni) [inline Tribes/src/app/dashboard/dashboard.component.css]: indentation is not ok here

File: Tribes/src/app/building-type/building-type.component.html [hunk 2]
```diff
@@ -17,8 +17,8 @@
           </div>
           <div *ngIf="towhalllevel !== 3" class="text-center mt-3">
             <button class="btn btn-warning"
-                [disabled]="getUpgardeingCost(townhallLevel) > goldAmount">Upgrade<br><small>for
-                  {{getUpgardeingCost(townhallLevel)}} gold</small></button>
+                [disabled]="getUpgradingCost(townhallLevel) > goldAmount">Upgrade<br><small>for
```
Comment (balazsbarni) [inline Tribes/src/app/building-type/building-type.component.html]: where we use this component?

File: Tribes/src/app/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -1,37 +1,111 @@
 <div class="container-center">
   <h1>DASHBOARD</h1>
   <h3>Click on one of the icons below to see the details</h3>
+  goldamount: {{goldAmount}}
+  number of farms: {{ numberOfFarms }}
+
```
Comment (etterke) [inline Tribes/src/app/dashboard/dashboard.component.html]: Why did you add these lines? What do you use this for?

File: Tribes/src/app/dashboard/dashboard.component.ts [hunk 4]
```diff
@@ -19,7 +19,7 @@ export class DashboardComponent implements OnInit {
   costOfNewBuilding = COST_NEW_BUILDING;
   numberOfFarms: number;
   numberOfMines: number;
-  numberOfBarracks: number;
+  numberOfBarracks : number;
```
Comment (etterke) [inline Tribes/src/app/dashboard/dashboard.component.ts]: There is an unnecessary space between s :

File: Tribes/src/app/building-type/building-type.component.html (hunk 2 above)
Comment (zsofi75) [reply Tribes/src/app/building-type/building-type.component.html]: Viola asked me to correct typo on this component. Please ask her about it.

File: Tribes/src/app/dashboard/dashboard.component.html [hunk 5]
```diff
@@ -1,36 +1,62 @@
...
   <h3>Click on one of the icons below to see the details</h3>
+
   <ul class="list-group list-group-horizontal-md">
     <li class="list-group-item" style="margin-left: 10px; margin-right: 50px">
       <a routerLink="townhall"><img src="assets/images/townhall.png"></a>
     </li>
-    <div *ngIf="numberOfBarracks === 0">
-      <li class="list-group-item">
-        <a class="inactive">
+
```
Comment (balazsbarni) [inline Tribes/src/app/dashboard/dashboard.component.html]: az ures sorok itt nem kellenek
